_pytest/deprecated.py

```diff
@@ -7,6 +7,11 @@
 """
 from __future__ import absolute_import, division, print_function
 
+
+class RemovedInPytest4_0Warning(DeprecationWarning):
```

Comment:
I think just `RemovedPytest4Warning` is enough, no need to add the `_0` in my opinion.